### PR TITLE
Add function to convert parquet schema to arrow schema

### PR DESCRIPTION
**Pull request by dryabinin-actionengine**

### CODE REVIEW

**Review by morphoix — APPROVED** [review]

File: modules/parquet/src/parquetjs/schema/types.ts [hunk 1]
```diff
@@ -409,3 +424,32 @@ function fromPrimitive_INTERVAL(value: any) {
 
   return {months, days, milliseconds: millis};
 }
+
+export const PARQUET_TYPE_MAPPING: {[type in ParquetType]: typeof DataType} = {
```
Comment (ibgreen) [inline modules/parquet/src/parquetjs/schema/types.ts:428]: I feel this mapping should go into a separate file e.g. `lib/convert-schema.ts`.

The `parquetjs` directory contains our fork of the parquet code and we should not add our additional logic there.

File: modules/parquet/src/parquetjs/utils/schema-converter-util.ts [hunk 2]
```diff
@@ -0,0 +1,47 @@
+import type {ParquetSchema} from '../schema/schema';
```
Comment (ibgreen) [inline modules/parquet/src/parquetjs/utils/schema-converter-util.ts:1]: Same here, let's move this from the `parquetjs` folder to the `lib` folder.

**Review by ibgreen — APPROVED** [review]

File: modules/parquet/src/parquetjs/schema/types.ts (hunk 1 above)
Comment (dryabinin-actionengine) [reply modules/parquet/src/parquetjs/schema/types.ts]: Moved to `lib/convert-schema.ts`

File: modules/parquet/src/parquetjs/utils/schema-converter-util.ts (hunk 2 above)
Comment (dryabinin-actionengine) [reply modules/parquet/src/parquetjs/utils/schema-converter-util.ts]: Moved to `lib/convert-schema.ts`

**Pull request merged**